core/base/document.php

```diff
@@ -1425,6 +1426,101 @@ public function __construct( array $data = [] ) {
 		parent::__construct( $data );
 	}
 
+	/**
+	 * Get document settings usage.
+	 *
+	 * @return array
+	 */
+	public function get_usage() {
+		$usage  = [];
```

Comment:
On recalc, check the notice: 
`Notice: Array to string conversion in /var/www/html/wp-content/plugins/elementor/includes/base/controls-stack.php on line 1465`

Reply:
https://github.com/elementor/elementor/pull/18687/commits/a35c31f09cb41bafa16cfadff676e7c18c859b07 - 
it caused by multidimensional array